### DISCUSSION TITLE
How to connect to UDP and TCP port from outside of cluster using nginx ingress

### DISCUSSION THREAD

**Discussion by Nish0808** [nginx/kubernetes-ingress#1918] [Q&A]
I installed the nginx 1.12.0 using Helm charts where I enabled globalConfiguration.create: true and provided the listeners and also installed transport-server-tcp.yaml and transport-server-udp.yaml. Now, I am wondering, after doing all these, how can I simply reach to my endpoint ( UDP and TCP ) from outside the cluster using nginx ingress controller. I have created Service type for nginx controller as NodePort and trying to access my app using (One of worker node IP):5000  where 5000 is my udp & tcp port but it fails and that is obvious as I do not see any nodePort opened for 5000? 

Also, is there a way to provide session affinity based on client IP to redirect all the traffic (UDP and TCP)from same client to one  pod only ?

**Comment by jasonwilliams14** [comment]
@Nish0808 can you post the output of your requests failing, with as much verbosity as possible?
Can you also show us the transport.yaml file you configured so we can review?
Thank you.

**Comment by jjngx** [comment]
Outdated. Waiting 4 years.
